app/Http/Controllers/IncidentController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Incident;
+use App\Resources\IncidentResource;
+use Carbon\Carbon;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Laravel\Lumen\Routing\Controller;
+
+class IncidentController extends Controller
+{
+    public function active(Request $request): JsonResponse
+    {
+        $all = $request->get('all');
+
+        if($all){
+            $incidents = Incident::isActive()->get();
+
+        } else {
+            $incidents = Incident::isActive()->isFire()->get();
+        }
+
+        return new JsonResponse([
+            'success' => true,
+            'data' => IncidentResource::collection($incidents),
+        ]);
+    }
+
+    public function search(Request $request): JsonResponse
+    {
+        //@todo validate date format
+        $day = $request->get('day');
+        if(!$day){
```

Comment:
It would probably be interesting to use Form Requests to validate all required data, no?